tutorials/deploy-nodejs-application-kyma/deploy-nodejs-application-kyma.md

```diff
@@ -0,0 +1,353 @@
+---
+title: Deploy a Node.js Application in the Kyma Runtime
+description: Build a basic Node.js application into OCI image and push it into Docker registry. Besides, describe the corresponding Kubernetes objects for the application. Based on the above, deploy the application into the Kyma runtime.
+auto_validation: true
+time: 25
+tags: [ tutorial>beginner, programming-tool>node-js]
+primary_tag: software-product>sap-btp\, kyma-runtime
+---
+
+## Prerequisites
+- You have finished the tutorial [Create a Basic Node.js Application with Express Generator](basic-nodejs-application-create)
```

Comment:
add period

Reply:
May I know what do you mean "period"?

Reply:
> add period

https://github.com/sap-tutorials/Tutorials/pull/18287#discussion_r840319445

Reply:
> add period
May I know wahy do you mean for "period"?

Reply:
> add period

May I know what do you mean by "period" here?

Reply:
This: .
;-)